faqs/emergency-shutdown.md

```diff
@@ -0,0 +1,130 @@
+
+## Do Dai holders need to redeem collateral, or is it just CDP owners?
+
+Both CDP owners and Dai holders need to redeem collateral. However, only Dai holders can redeem ETH immediately after the Emergency Shutdown is triggered. CDP owners need to wait the 6 hour time period.
+
+## What do I do in the case of an Emergency Shutdown if I'm a PETH holder who does not own a CDP?
+
+The PETH ratio drops instantaneously when an Emergency Shutdown is triggered and comes back up as CDPs are processed. PETH holders should wait the 6 hour delay period like CDP owners.  
+
+## If the prices keep falling after the price feeds are frozen, doesn't this mean that people redeeming Dai and CDPs may get less than $1 of collateral?
+
+Yes, it is possible for users to get less than a dollar worth of collateral. Dai holders are able to immediately redeem their Dai for collateral and may experience some slippage. CDP owners must wait the 6 hour delay period for claims to be calculated and are therefore exposed to the risk of the underlying collateral price moving during these 6 hours.
+
+## How quickly can an Emergency Shutdown be performed?
+
+To trigger an Emergency Shutdown, an Executive Vote needs to be voted on by MKR voters. The proposal needs to garner enough MKR until it beats the previous proposal, in line with [continuous approval voting](https://github.com/makerdao/community/blob/master/faqs/governance.md#what-is-continuous-approval-voting). Therefore, it depends on how quickly MKR voters can act to beat the previous winning proposal.
+
+## What happens to MakerDAO after an Emergency Shutdown?
+
+After Emergency Shutdown is triggered, the collateral redemption process takes place. In the meantime, it is up to anyone whether they want to redeploy the system or not, and how they would deploy it.
+
+## Who can redeploy the system?
+
+Anyone can redeploy the system since it is open source software.
+
+## What if multiple redeployments happen, who decides which one is legitimate?
+
+Depending on the details of each redeployment, the market, along with various system participants will have to choose the one with the most appropriate changes. Some factors include:
+
+-   No unwarranted code changes
+
+-   Changes in the distribution of the MKR token
+
+-   Changes in the Risk Parameters
+
+## Who decides how the system should be redeployed?
+
+Since it is open source software, anyone can decide. Ideally, the parameters of redeployment should depend on the reason for the Emergency Shutdown, and should not be altered unilaterally and arbitrarily. Here is a rough example of a framework for making changes on redeployment:
+
+| Reason            | Solution                                                                                                                                                                        |
```

Comment:
You don't need to space tables manually. 

```
| Reason | Solution |
|---|---|
| Governance attack | Fork out malicious MKR holders in new redistribution,  redeploy system with everything else as-is|
```

Reply:
autogenerated the table